sdk/src/client/secret/types.rs

```diff
@@ -32,6 +32,19 @@ pub struct StrongholdDto {
     /// The path for the Stronghold file
     pub snapshot_path: String,
 }
+
+/// Represents the Stronghold password.
```

Comment:
Do we need to mention Stronghold? could also be used for the SEED_1 we still have somewhere. Its just a password wrapper

Reply:
hmm, but isn't a Password something else than a Seed?